src/views/virtualmachinesinstances/hooks/useVirtualMachinesInstancesColumns.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  K8sResourceCommon,
+  TableColumn,
+  useActiveColumns,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+const useVirtualMachinesInstancesColumns = (): { title: string; id: string }[] => {
+  const { t } = useTranslation('plugin__kubevirt-plugin');
```

Comment:
I guess this will be repeated a lot in KubeVirt plugin codebase, we can create a custom hook like `useKubevirtTranslation`

```ts
const { t } = useKubevirtTranslation();
```

Also not sure what's the actual CamelCase convention for the word KubeVirt, but I think most people spell it as Kubevirt with lowercase "v" in the codebase :smiley:

Reply:
+1

Reply:
I actually thought about it, i was afraid that if ill add it people wont be aware of it and still use useTranslation('plugin__kubevirt-plugin'); 

I looked for a way to bind it directly into i18n but couldn't find a simple way. you do know any simple way to change i18n default namespace to kubevirt in i18n setting? so it will apply only to our plugin?